packages/storybook/stories/CallComposite/JoinExistingCall.stories.tsx

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CallComposite } from '@azure/communication-react';
+import { Stack } from '@fluentui/react';
+import { Meta } from '@storybook/react/types-6-0';
+import React, { useRef } from 'react';
+import { COMPOSITE_FOLDER_PREFIX, compositeExperienceContainerStyle } from '../constants';
+import { getDocs } from './CallCompositeDocs';
+import { ContosoCallContainer } from './snippets/Container.snippet';
+import { ConfigJoinCallHintBanner } from './snippets/Utils';
+
+const JoinExistingCallStory: (args) => JSX.Element = (args) => {
+  const controls = useRef({
+    callLocator: args.callLocator,
+    userId: args.userId,
+    token: args.token,
+    displayName: args.displayName,
+    callInvitationURL: args.callInvitationURL
+  });
+
+  const areAllKnobsSet =
+    !!controls.current.callLocator &&
+    !!controls.current.userId &&
+    !!controls.current.token &&
+    !!controls.current.displayName;
+  return (
+    <Stack horizontalAlign="center" verticalAlign="center" styles={compositeExperienceContainerStyle}>
+      {areAllKnobsSet ? (
+        <ContosoCallContainer
+          locator={controls.current.callLocator}
+          userId={{ communicationUserId: controls.current.userId }}
+          token={controls.current.token}
+          displayName={controls.current.displayName}
+          callInvitationURL={controls.current.callInvitationURL}
+        />
+      ) : (
+        <ConfigJoinCallHintBanner />
+      )}
+    </Stack>
+  );
+};
+
+export const JoinExistingCall = JoinExistingCallStory.bind({});
+
+export default {
+  id: `${COMPOSITE_FOLDER_PREFIX}-call-joinexistingcall`,
+  title: `${COMPOSITE_FOLDER_PREFIX}/CallComposite/Join Existing Call`,
+  component: CallComposite,
+  argTypes: {
+    callLocator: { control: 'text', defaultValue: '', name: 'Call locator (ACS group ID or Teams meeting link)' },
```

Comment:
Is it possible to set placeholder text here? Instead of "Adjust string dynamically" 
![image](https://user-images.githubusercontent.com/2684369/128103311-680a8789-d3ab-49e9-a088-b8f15a931151.png)

(If its possible, doesn't have to updated in this PR, more just curious but if possible would be nice for us to update it at some point)